File: solutions/csharp/raindrops/1/Raindrops.cs
using System;

public static class Raindrops
{
    public static string Convert(int number)
    {
        var rain = "";

        rain += number % 3 == 0 ? "Pling" : "";
        rain += number % 5 == 0 ? "Plang" : "";
        rain += number % 7 == 0 ? "Plong" : ""; 
        rain += rain.Equals("") ? number.ToString() : "";

        return rain;
    }
}

File: solutions/kotlin/raindrops/1/src/main/kotlin/Raindrops.kt
object Raindrops {

    fun convert(n: Int): String {
        var result = "";
            
    	result += if (n % 3 == 0) "Pling" else ""
    	result += if (n % 5 == 0) "Plang" else ""
    	result += if (n % 7 == 0) "Plong" else ""
    
    	if (result == "") return n.toString()
    
    	return result
    }
}
